src/raster/buffer.rs

```diff
@@ -142,24 +148,48 @@ impl<T: GdalType + Copy> From<Array2<T>> for Buffer<T> {
 #[cfg(test)]
 mod tests {
     use crate::raster::Buffer;
-    use ndarray::Array2;
+    use ndarray::{Array2, ShapeBuilder};
 
     #[test]
     fn convert_to() {
         let b = Buffer::new((5, 10), (0..5 * 10).collect());
         let a = b.clone().to_array().unwrap();
+        let expected = Array2::from_shape_fn((10, 5), |(row, col)| row as i32 * 5 + col as i32);
+        assert_eq!(a, expected);
         let b2: Buffer<_> = a.into();
         assert_eq!(b, b2);
     }
 
     #[test]
     fn convert_from() {
-        let a = Array2::from_shape_fn((10, 5), |(y, x)| y as i32 * 10 + x as i32);
+        let a = Array2::from_shape_fn((10, 5), |(row, col)| row as i32 * 5 + col as i32);
```

Comment:
`shapes` uses `(y, x)`, do we pick one?

Reply:
I only changed the binding names to check my sanity. Want me to change it back?

Reply:
Just thinking we should be consistent about them.